### PR TITLE
Port builder section rework

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/263

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

redirects:
https://github.com/PureStake/moonbeam-mkdocs/pull/75

removes images:
https://github.com/PureStake/moonbeam-docs/pull/279

### CODE REVIEW

File: builders/build/eth-api/dev-env/truffle.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+# 使用Truffle部署至Moonbeam
+
+<style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/RD5MefSPNeo' frameborder='0' allowfullscreen></iframe></div>
+<style>.caption { font-family: Open Sans, sans-serif; font-size: 0.9em; color: rgba(170, 170, 170, 1); font-style: italic; letter-spacing: 0px; position: relative;}</style>
+
+## 概览 {: #introduction }
+
+本教程将向您展示通过以太坊上常用的智能合约开发工具[Truffle](https://www.trufflesuite.com/){target=blank}将基于Solidity智能合约部署至Moonbeam节点的操作过程。鉴于Moonbeam兼容以太坊的特性，Truffle可与Moonbeam节点一起直接使用。
+
+为简化使用Truffle的步骤，您可以使用[Moonbeam Truffle box](https://github.com/PureStake/moonbeam-truffle-box){target=blank}。这将提供一个模板设置以加快在Moonbeam上的部署流程。Moonbeam Truffle box自带[Moonbeam Truffle plugin](https://github.com/purestake/moonbeam-truffle-plugin){target=blank}，使您可以快速开始使用[Moonbeam开发节点](/ builders/get-started/moonbeam-dev/){target=blank}。
```
Comment (eshaben) [inline builders/build/eth-api/dev-env/truffle.md:15]:
```suggestion
为简化使用Truffle的步骤，您可以使用[Moonbeam Truffle box](https://github.com/PureStake/moonbeam-truffle-box){target=blank}。这将提供一个模板设置以加快在Moonbeam上的部署流程。Moonbeam Truffle box自带[Moonbeam Truffle plugin](https://github.com/purestake/moonbeam-truffle-plugin){target=blank}，使您可以快速开始使用[Moonbeam开发节点](/builders/get-started/moonbeam-dev/){target=blank}。
```

File: builders/build/eth-api/dev-env/truffle.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+现在，您已经创建了一个简单的Truffle项目，接下来您可以启动本地Moonbeam开发节点部署合约。Moonbeam Truffle plugin提供了一种在后台使用[Docker](https://www.docker.com/){target=blank}快速开始开发节点的方法。
+
+要在您的本地环境启动Moonbeam开发节点，您需要：
+
+1. 下载对应的Docker镜像：
+
+    ```
+truffle run moonbeam install
+    ```
```
Comment (eshaben) [inline builders/build/eth-api/dev-env/truffle.md:79]:
```suggestion
    ```
    truffle run moonbeam install
    ```
```

File: builders/build/eth-api/dev-env/truffle.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+在本示例中，`8000000000000000000000000`是合约初始Token供应量，即800万小数点后18位。
+
+## 使用Truffle部署合约至Moonbeam {: #deploying-a-contract-to-moonbeam-using-truffle }
+
+在部署之前必须先对合约进行编译。友情提示，您将使用`migrations/1_initial_migration.js`脚本先部署`Migrations.sol`合约。这将为您启用Truffle的迁移功能。您可以执行以下步骤编译和部署您的合约：
+
+1. 编译合约：
+
+    ```
+truffle compile
```
Comment (eshaben) [inline builders/build/eth-api/dev-env/truffle.md:254]:
```suggestion
    truffle compile
```

File: builders/build/eth-api/libraries/ethersjs.md [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+    const provider = new ethers.providers.StaticJsonRpcProvider(
+      providerRPC.moonbase.rpc, 
+      {
+        chainId: providerRPC.moonbase.chainId,
+        name: providerRPC.moonbase.name,
+      }
+    );
+    ```
+
+=== "Moonbeam Dev Node"
```
Comment (eshaben) [inline builders/build/eth-api/libraries/ethersjs.md:127]: dev node usually gets translated here

File: builders/build/eth-api/libraries/web3py.md [hunk 5]
```diff
@@ -0,0 +1,495 @@
...
+
+=== "Moonbase Alpha"
+
+    ```python
+    from web3 import Web3
+    
+    web3 = Web3(Web3.HTTPProvider('{{ networks.moonbase.rpc_url }}'))
+    ```
+
+=== "Moonbeam Dev Node"
```
Comment (eshaben) [inline builders/build/eth-api/libraries/web3py.md:71]: this should be translated

File: builders/get-started/networks/index.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+title: Networks
```
Comment (eshaben) [inline builders/get-started/networks/index.md:2]: can this be translated?

File: builders/integrations/.pages [hunk 7]
```diff
@@ -4,3 +4,4 @@ nav:
  - bridges
  - indexers
  - oracles
+ - wallets
```
Comment (eshaben) [inline builders/integrations/.pages:7]: couldn't add a comment on the actual line, but is the translation for integrations the same as integrate?

File: builders/integrations/indexers/subquery.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+2. 创建您SubQuery项目的目录：
+
+    ```
+    mkdir subquery-example && cd subquery-example
+    ```
+    
+3. 使用下方命令以初始化您SubQuery项目：
+
+    ```
+subql init PROJECT_NAME
```
Comment (eshaben) [inline builders/integrations/indexers/subquery.md:46]:
```suggestion
    subql init PROJECT_NAME
```

File: builders/integrations/wallets/index.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+---
+title: Integrate Wallets
```
Comment (eshaben) [inline builders/integrations/wallets/index.md:2]: can this be translated?

File: tokens/manage/index.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+---
+title: Account Management
```
Comment (eshaben) [inline tokens/manage/index.md:2]: can this be translated?

File: builders/get-started/networks/index.md (hunk 6 above)
Comment (eshaben) [reply builders/get-started/networks/index.md:2]: never mind this comment

File: builders/integrations/wallets/index.md (hunk 9 above)
Comment (eshaben) [reply builders/integrations/wallets/index.md:2]: never mind

File: tokens/manage/index.md (hunk 10 above)
Comment (eshaben) [reply tokens/manage/index.md:2]: never mind

**Comment by eshaben** [comment]
Oh! One more thing I forgot about... is there a separate PR for the english repo to remove the images? (all the ones that need to be removed are listed on the original PR)

File: builders/integrations/.pages (hunk 7 above)
Comment (hyd628) [reply builders/integrations/.pages:7]: Yes, they are the same word in Chinese.

File: builders/build/eth-api/dev-env/hardhat.md [hunk 11]
```diff
@@ -0,0 +1,316 @@
...
+
+// 2. Import your private key from your pre-funded Moonbase Alpha testing account
+const { privateKey } = require('./secrets.json');
+
+module.exports = {
+  // 3. Specify the Solidity version
+  solidity: "0.8.1",
+
+  networks: {
+    // 4. Add the Moonbase Alpha network specification
```
[the remark this answers is not in the document]
Comment (hyd628) [reply builders/build/eth-api/dev-env/hardhat.md:152]: hmm, I think English comments in the sample code is fine. There is a chance of some editors not supporting unicode characters, and some languages might require specific syntax for unicode characters (like escapes), which might create inconsistencies and compile errors that could be hard to detect. 

Let me know what you think. :D

**Comment by hyd628** [comment]
Thanks for the feedback @eshaben . Incorporated most of the feedback and created a PR to delete the extra images.

File: builders/build/eth-api/dev-env/hardhat.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/build/eth-api/dev-env/hardhat.md:152]: ok sounds good! 👍

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**